providers/salesforce/read.go

```diff
@@ -67,5 +67,9 @@ func makeSOQL(config common.ReadParams) string {
 		soql.Where("IsDeleted = true")
 	}
 
+	if config.Filter != "" {
+		soql.Where(config.Filter)
+	}
```

Comment:
Does SOQL have the equivalent of SQL injection? This seems like a serious possibility here. I'm gonna approve this since Ampersand controls the contents of this field but this is a concern for me.

Reply:
They have a section on this (https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/pages_security_tips_soql_injection.htm) and their advice is basically to stick to static queries. I am going to add a comment here to remind us to be aware of this.